Quadriga/src/main/java/edu/asu/spring/quadriga/domain/workspace/ITextFile.java

```diff
@@ -29,15 +29,16 @@
     public abstract String getFileContent();
 
     public abstract void setFileContent(String fileContent);
-    
+
     public abstract String getTextFileURI();
 
     /**
+     * This method sets the uri of the related TextFile based on the passed to
+     * the method.
+     * 
      * @param uriPrefix
-     *      Prefix of the URI specified in the properties file.
-     *      eg: http://quadriga.io/text/textid
+     *            Prefix of the URI specified in the properties file.
```

Comment:
this method doesn't care where the prefix is coming from. All it cares about is that it gets the base URI/prefix for generating uris.